README.md

```diff
@@ -1,65 +1,95 @@
 # Desktop Embedding for Flutter
 
 This project was originally created to develop Windows, macOS, and Linux
-implementations of [Flutter](https://github.com/flutter/flutter). That work has
-since become part of the
-[Flutter engine repository](https://github.com/flutter/engine), and this
+embeddings of [Flutter](https://github.com/flutter/flutter). That work has
+since [become part of
+Flutter](https://github.com/flutter/flutter/wiki/Desktop-shells), and this
 project is now just an example of, and test environment for, building
-applications using those libraries.
+applications using those libraries in their current state.
```

Comment:
Would be great to also mention the experimental plugins.

Reply:
Done.